mapper/mapper.go

```diff
@@ -268,6 +273,8 @@ func CreateListSelectorPage(name string, selectedValues []filter.DimensionOption
 func CreateRangeSelectorPage(name string, selectedValues []filter.DimensionOption, allValues dataset.Options, filter filter.Model, dst dataset.Model, datasetID, releaseDate string) rangeSelector.Page {
 	var p rangeSelector.Page
 
+	log.Debug("mapping api response models to range selector page model", log.Data{"filterID": filter.FilterID, "datasetID": datasetID})
```

Comment:
Would name be useful here? gives context for which specific area inside a dataset is causing issues